tests/integration/cloudformation/test_template_engine.py

```diff
@@ -472,30 +472,86 @@ def test_import_values_across_stacks(self, deploy_cfn_template, s3_client, cfn_c
         # assert cfn_client.list_imports(ExportName=export_name)["Imports"]
 
 
-@pytest.mark.skip(reason="Macros not yet supported")
 class TestMacros:
     @pytest.mark.aws_validated
-    def test_global_scope(
+    def test_macro_deployment(
         self, deploy_cfn_template, cfn_client, create_lambda_function, lambda_client, snapshot
+    ):
+        macro_function_path = os.path.join(
+            os.path.dirname(__file__), "../templates/macros/format_template.py"
+        )
+        macro_name = "SubstitutionMacro"
+        stack_with_macro = create_macro(
+            macro_name,
+            macro_function_path,
+            deploy_cfn_template,
+            create_lambda_function,
+            lambda_client,
+        )
```

Comment:
I'd suggest removing this utility (`create_macro`) actually. It doesn't save that many lines of code and is just another indirection hiding what is actually  provisioned.